### DISCUSSION TITLE
Check url if Encryption key is set/correct

### DISCUSSION THREAD

**Discussion by pkuehnel** [tobiasehlert/teslamateapi#203] [Q&A]
Is there any endpoint to check if the encryption key is properly set? Or any other option to check without waking up the car? I am using TeslaMateApi in [TeslaSolarCharger](https://github.com/pkuehnel/TeslaSolarCharger) and implemented a check for configuration issues which autoruns when frontend is open on at least one client. Currently I am using `/api/v1/cars` if this crashes (or returns an empty string) I display an information that something is wrong with TeslaMateApi. But as `/api/v1/cars` does not rely on the encryption key it does not crash and any endpoint relying on it, potentially wakes up the car / keeps it awake.

**Comment by LelandSindt** [comment]
``` Is there any endpoint to check if the encryption key is properly set?``` --- there is no /verify_encryption_key endpoint. 

Do I understand correctly that you want regularly/continuously check that  the provided `ENCRYPTION_KEY` can decrypt the token managed by teslamate? Can you help me better understand your usecase?

**Reply by pkuehnel** [reply]
I just want to detect configuration issues, so if anything is not working I want to display why.
